net-mgmt/zabbix-agent/src/opnsense/mvc/app/models/OPNsense/ZabbixAgent/Menu/Menu.xml

```diff
@@ -0,0 +1,12 @@
+<menu>
+    <Services>
+      <ZabbixAgent VisibleName="Zabbix Agent" cssClass="fa fa-certificate fa-fw">
+          <Settings order="10" url="/ui/zabbixagent/">
+              <MainSettings VisibleName="Main Settings" url="/ui/zabbixagent/#subtab_zabbixagent-settings-main"/>
```

Comment:
No no, it's not in the menu, it's just the internal structure. In the GUI it's just `Services -> Zabbix Agent -> Settings` and on the settings page some tabs represent the 4th level.